pandas/__init__.py

```diff
@@ -180,7 +180,7 @@
 import pandas.arrays
 
 # use the closest tagged version if possible
-from ._version import get_versions
+from pandas._version import get_versions
```

Comment:
is this added automatically by versioneer?

Reply:
yes, thanks for spotting this - I've updated `versioneer.py` to use an absolute import